packages/cf-component-icon/test/Icon.js

```diff
@@ -40,4 +40,12 @@ describe('Icon', function() {
       <i className="cf-icon cf-icon--info-sign"></i>
     );
   });
+
+    it('should render role', function() {
+    assertEqualJSX(
```

Comment:
@thejameskyle indentation adds so many wasted bytes to a git clone. if we remove all indentation cloning will be much faster.